charts/nfdiv-cms/values.yaml

```diff
@@ -6,7 +6,7 @@ java:
         IDAM_API_BASEURL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net"
         AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
         IDAM_API_REDIRECT_URL: "https://div-pfe-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/authenticated"
-        CASE_DATA_STORE_BASEURL: "http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        CASE_DATA_STORE_BASEURL: "http://ccd-data-store-api-nfdiv-ccd-definitions-pr-37.service.core-compute-preview.internal"
```

Comment:
for pr update preview yaml with this `http://ccd-data-store-api-nfdiv-ccd-definitions-pr-37.service.core-compute-preview.internal`